src/interfaces/events/types.ts

```diff
@@ -0,0 +1,18 @@
+import type { BaseMessage } from '../../core/types.js';
+import type { BaseMessageReply } from '../../core/message-reply.js';
+import type { Event } from '../../event-log/event-log.js';
+import type { DwnInterfaceName, DwnMethodName } from '../../core/message.js';
+
+export type EventsGetDescriptor = {
+  interface : DwnInterfaceName.Events;
+  method: DwnMethodName.Get;
+  watermark?: string;
+};
+
+export type EventsGetMessage = BaseMessage & {
+  descriptor: EventsGetDescriptor;
+};
+
+export type EventsGetReply = BaseMessageReply & {
```

Comment:
rather than create a new subtype, should we just add something like `events?: Event[];` to the `MessageReply`?

otherwise how will a developer reason about "what is the return type of `dwn.processMessage(tenant, message)`"?  are we expecting them to forcibly `as EventsGetReply` in order to get any sort of typing support?

Reply:
currently yeah. that, or remain consistent with what henry's done for `RecordsRead` which is add a specific handler for that in `dwn.ts`